packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -527,6 +529,7 @@ export async function downloadDocsAndUpsertFHIR({
               };
             }
             file = await uploadToS3();
+            if (!file) return;
```

Comment:
I think we can't just return - we need to account for the download and conversion counters. Maybe just throw `NotFoundError` and check this type before reporting to Sentry - if its a 404 we just log and don't report?